srcpkgs/breeze-cursors/template

```diff
@@ -1,11 +1,9 @@
 # Template file for 'breeze-cursors'
 pkgname=breeze-cursors
 version=1.0
-revision=1
+revision=2
 create_wrksrc=yes
-archs=noarch
```

Comment:
Why? Are there plans to drop noarch soon?

Reply:
Mostly because it would be a noarch package depending directly on an arch package. Now that I think of it, that's also a lot of python packages. Should I leave noarch, then?

Reply:
Let it stay, breeze-snow-cursor-theme can be noarch too.

Reply:
Done.